include/nbl/builtin/glsl/ext/RadixSort/radix_sort.glsl

```diff
@@ -0,0 +1,69 @@
+#ifndef _NBL_GLSL_EXT_RADIXSORT_INCLUDED_
+#define _NBL_GLSL_EXT_RADIXSORT_INCLUDED_
+
+#define _NBL_GLSL_SCRATCH_SHARED_SIZE_DEFINED_ (_NBL_GLSL_WORKGROUP_SIZE_ * 2 * 4)
```

Comment:
what are the 2 and 4 constants ?

Reply:
2 * `_NBL_GLSL_WORKGROUP_SIZE_` is required for the following piece of code (for `default_histogram.comp` exclusively) which is currently present in `nbl_glsl_ext_RadixSort_workgroupSort`
```
scratch_shared[scatter_idx] = data.x;
	scratch_shared[_NBL_GLSL_WORKGROUP_SIZE_ + scatter_idx] = data.y;
	barrier();

	data = uvec2(scratch_shared[gl_LocalInvocationIndex], scratch_shared[gl_LocalInvocationIndex + _NBL_GLSL_WORKGROUP_SIZE_]);
```

`4` was because I erroneously thought I have to convert it to bytes